inc/lift/RequestHandle.h

```diff
@@ -191,14 +202,16 @@ class RequestHandle
      * @param request_pool The request pool that generated this handle.
      * @param curl_handle  The CURL* handle for this Request.
      * @param curl_pool    The CurlPool to return the CURL* handle when this request destructs.
+     * @param max_download_bytes    The maximum number of bytes to download, if -1, will download entire file.
```

Comment:
on_complete_handler? I know it isn't from your code change but it was missing from the doc blocks previously

Reply:
Comment added